project.clj

```diff
@@ -17,4 +17,6 @@
    :1.9  {:dependencies [[org.clojure/clojure "1.9.0"]
                          [ring/ring-spec "0.0.4"]]}
    :1.10 {:dependencies [[org.clojure/clojure "1.10.0"]
+                         [ring/ring-spec "0.0.4"]]}
+   :1.12 {:dependencies [[org.clojure/clojure "1.12.0"]
```

Comment:
Missing 1.11 profile, and the 1.12 profile is not included in the "test-all" alias.

Reply:
Added missing 1.11 profile
Added both 1.11  and 1.12 to the test-all alias